### DISCUSSION TITLE
Can we create a taxonomy of Open Source together?

### DISCUSSION THREAD

**Discussion by gyehuda** [finos/open-source-readiness#11] [General]
Seems to me it might help to clarify the spectrum of project types that people consider to be Open Source. For example, we talk about using, contributing, or considering the risks of "open source" as a whole. But there are differences between things like small Node.JS modules declared as a project dependency, open source tools and scripts used by developers, and  vendor products that happen to have open source versions, etc.  They are all open source, in a sense, but the concerns are so different that is might be worth decomposing the term into categories.

I suggest this might help with readiness activities since an entity might want a document to focus on one category at a time.

Would participants like to suggest different categories of open source projects? Thanks.

**Comment by vmbrasseur** [comment]
This sounds a lot like [Open Source Archetypes: A Framework For Purposeful Open Source](https://opentechstrategies.com/archetypes) from OTS.
